drivers/display/ssd1327.c

```diff
@@ -482,6 +498,9 @@ static DEVICE_API(display, ssd1327_driver_api) = {
 #define SSD1327_DEFINE_MIPI(node_id)                                                               \
 	static uint8_t conversion_buf##node_id[SSD1327_CONV_BUFFER_SIZE(node_id)];                 \
 	static struct ssd1327_data data##node_id;                                                  \
+	COND_CODE_1(DT_NODE_HAS_PROP(node_id, grayscale_table), (                                  \
+	static uint8_t ssd1327_grayscale_table_##node_id[SSD1327_SET_LUT_COUNT] =                  \
+	DT_PROP(node_id, grayscale_table);), ())                                                   \
```

Comment:
For saving 15 bytes of ram, would this be `static const uint8_t ssd1327_grayscale_table_`?